packages/video_player/video_player/pubspec.yaml

```diff
@@ -1,7 +1,7 @@
 name: video_player
 description: Flutter plugin for displaying inline video with other Flutter
   widgets on Android and iOS.
-version: 0.10.7
+version: 0.10.8
```

Comment:
I think we can keep this as 0.10.7+1 since there are no public api changes.
https://dart.dev/tools/pub/versioning#semantic-versions

Reply:
I guess this is a grey area but this is a significant enough change and represents a new feature (support for add-to-app) so I think an increment of the third number is better.

Reply:
SG!